src/feasibility_residual.jl

```diff
@@ -5,11 +5,16 @@ export FeasibilityResidual
 A feasibility residual model is created from a NLPModel of the form
 
     min f(x)
-    s.t c(x) = 0
+    s.t cℓ ≤ c(x) ≤ cu
+        bℓ ≤   x  ≤ bu
 
-by defining the function F(x) = c(x). If the problem has
-bounds on the variables or more constraints, an error
-is thrown.
+by creating slack variables s and defining F(x,s) = c(x) - s. The resulting NLS problem is
+
+    min ¹/₂‖c(x) - s‖²
```

Comment:
It seems f(x) disappeared?!

Reply:
Yes, `f` is ignored on for the NLS model. I could change to `min 0`.